lighthouse-core/fraggle-rock/user-flow.js

```diff
@@ -108,20 +109,60 @@ class UserFlow {
    */
   async navigate(requestor, stepOptions) {
     if (this.currentTimespan) throw new Error('Timespan already in progress');
+    if (this.currentNavigation) throw new Error('Navigation already in progress');
 
     const options = this._getNextNavigationOptions(stepOptions);
     const gatherResult = await navigationGather(requestor, options);
 
     this._addGatherStep(gatherResult, options);
+  }
+
+  /**
+   * @param {StepOptions=} stepOptions
+   */
+  async startNavigation(stepOptions) {
```

Comment:
add a comment describing that this is an alternative to the `.navigate()` method? and perhaps linking to the docs you  added?